prod/westeurope/siem/event_hub/terragrunt.hcl

```diff
@@ -16,17 +16,37 @@ terraform {
 }
 
 inputs = {
-  name                = "siem"
+  namespce_name       = "siem"
```

Comment:
typo?

Reply:
Yes, sorry.
It has been fixed within the module **azurerm_eventhub**
I've assumed in the rel v2.0.15 which needs to be create.